src/utils/stylingpanel-style-formatter.js

```diff
@@ -0,0 +1,195 @@
+import colorUtils from './color-utils';
```

Comment:
So I don't see why you need to add this file. As I understand it, the only thing that this PR should do is move options to the new styling panel. That means that the styling should be applied the same way as before. And if there needs to be changes in the style formatter, they should be in the `style-formatter` file behind a flag, not in a new file

Reply:
If the existing style formatter file is to be used, there are multiple places where we would need the feature flag check and in some places this would make the existing code messy and probably lead to errors. So to avoid breaking the existing code accidently we have kept the styling panel implementation in new file.